hw/vendor/pulp_riscv_dbg/debug_rom/gen_rom.py

```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
This is a vendored-in file, changes need to go upstream or in a patch.

Reply:
Thanks for catching this one. Looks like this script is used in a standalone way, although it does not have a main() function entry. Hence no need to remove the shebang. I've reverted this change.